zipline/utils/security_list.py

```diff
@@ -0,0 +1,134 @@
+import os.path
+import pytz
+import pandas as pd
+from datetime import datetime
+from os import listdir
+
+DATE_FORMAT = "%Y%m%d"
+import zipline
+zipline_dir = os.path.join(*zipline.__path__)
+SECURITY_LISTS_DIR = os.path.join(zipline_dir, '..', 'security_lists')
+
+
+def loopback(symbol, *args, **kwargs):
+    return symbol
+
+
+class SecurityListSet(object):
+
+    def __init__(self, current_date_func, lookup_func=None):
+        if lookup_func is None:
+            self.lookup_func = loopback
+        else:
+            self.lookup_func = lookup_func
+        self.current_date_func = current_date_func
+        self._leveraged_etf = None
+
+    @property
+    def LEVERAGED_ETF_LIST(self):
```

Comment:
BECAUSE I WAS THINKING OF IT LIKE AN ENUM. But, that's not right, so I can drop the caps :).